### PR TITLE
man: systemd-timesyncd.service(8)

### LINKED ISSUE #5735: systemd-timesyncd manual should explain it uses SNTP, not NTP
### Submission type

  - [ ] Bug report
  - [x] Request for enhancement (RFE)

### systemd version the issue has been seen with

> 215, 226, git master of Apr 12, 2017

### Used distribution

> Gentoo, Debian Jessie

The manual page for systemd-timesyncd does not mention anything about the way it adjusts time. Naturally I assumed it would gracefully slew the clock, but after searching through the issues, I found [a single mention](https://github.com/systemd/systemd/issues/2893#issuecomment-210374428) that it intentionally implements only SNTP, not NTP. It is never even intended to adjust the system clock gracefully.

As I had never heard of SNTP before, I looked it up. SNTP is said to be inferior to NTP:
https://www.meinbergglobal.com/english/faq/faq_37.htm
http://www.timetools.co.uk/2013/07/23/the-difference-between-ntp-and-sntp-protocols/

That makes it crucial that the user must be aware of timesyncd using _only_ SNTP. **This should be mentioned in the manual page** - I don't know where else I should have looked.

If you wish, it would also be good to explain a little bit about the shortcomings of SNTP, and perhaps even suggest to use a sophisticated NTP client instead if the shortcomings might be problematic.

### PR DESCRIPTION
**Pull request by jcpunk**
Updates the documentation to resolve #5735

### CODE REVIEW

File: man/systemd-timesyncd.service.xml [hunk 1]
```diff
@@ -64,6 +64,12 @@
     reboots to ensure it monotonically advances even if the system
     lacks a battery-buffered RTC chip.</para>
 
+    <para>The <filename>systemd-timesyncd</filename> service
+    specifically implements only SNTP. This minimalistic
+    service will generally set the system clock rather
+    than slowly adjust it. More complex use cases are not
+    covered by <filename>systemd-timesyncd</filename>.</para>
```
Comment (keszybz) [inline man/systemd-timesyncd.service.xml]: That's not true actually. Please see `manager_adjust_clock` — the kernel should slew the clock for adjustments smaller than 0.5 s.

Comment (jcpunk) [reply man/systemd-timesyncd.service.xml]: Interesting!  I'll get this fixed up.

**Comment by jcpunk** [comment]
Updated per feedback

**Comment by keszybz** [comment]
Thanks.

**Pull request merged**